### ISSUE TITLE
bhavcopy ZIP file format change issue

### ISSUE THREAD

**Issue by gaurav34065** [sdabhi23/bsedata#59]
Looks like in [here](https://github.com/sdabhi23/bsedata/blob/dec410b1e166085ef45f54f66078acdde209ab24/bsedata/bhavcopy.py#L16C9-L16C76) current format we're fetching as 

https://www.bseindia.com/download/BhavCopy/Equity/EQ230924_CSV.ZIP

Needs to be updated like
https://www.bseindia.com/download/BhavCopy/Equity/BSE_EQ_BHAVCOPY_23092024_T0.ZIP

**Comment by sdabhi23** [comment]
hye @gaurav34065 is this change on BSE's end breaking the bhavcopy feature?

**Comment by gaurav34065** [comment]
@sdabhi23 yes, seems like it's not able to find the bhavcopy. I tested this via below sample code
```
#!/bin/env python3

from bsedata.bse import BSE
from datetime import datetime, timedelta

b = BSE()
b = BSE(update_codes = True)

yesterday = datetime.now().date() - timedelta(days=1)
q = b.getBhavCopyData(yesterday)
print(q)
```
Getting below exception
`bsedata.exceptions.BhavCopyNotFound: The BhavCopy file was not found on the BSE website. You are probably trying to get data for a trading holiday.`

**Comment by sdabhi23** [comment]
Oh okay, I will try to release a fix sometime next week. Thanks for reporting!
